### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by arueda**
Please focus on cpp and h files, since documentation files were autogenerated by doxygen.
Please raise any concerns about how methods are named or how the clases are being used.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include "Engine.h"
+
+int main( int argc, char* argv[] )
+{
+
```
Comment (marek2901) [inline src/main.cpp]: Style thing, but those spaces are probably unnecessary.

File: src/Game.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#include "Game.h"
+
+
+Game::Game():gameType(Game::TEXT){
+}
+
+bool Game::loadGameFromFile(const char* gameFileName){
```
Comment (marek2901) [inline src/Game.cpp]: Inconsistent brackets style. Below is different than here

File: src/Engine.cpp [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+void Engine::init(int argc, char* argv[])
+{
+	if( argc < 2){
+		logger.print(Errors::USAGE);
+	}else{
+		game.loadGameFromFile(argv[1]);
+	}
+	
+	/*int result = SDL_Init( SDL_INIT_VIDEO ) ;
```
Comment (marek2901) [inline src/Engine.cpp]: I can see plenty commented code here. Not sure if it is done on purpose or just posted by mistake (dead code).

**Comment by arueda** [comment]
@marek2901 I updated the code, addressing the changes that you suggested.
Regarding the style used for curly braces this is what I want to achieve:

Method definition.
```
returnType MehodName (arguments) 
{
    //method definition
}
```
Class declaration
```
class className 
{
}
```
if - else
```
if (condition) {
} else {
}
```
Loops
```
while (condition) {
}
```

**Comment by arueda** [comment]
@marek2901 Something funny happened to the commit. Looks like the raspberry pi where the code is being developed has an incorrect time set on the clock and the commits look like two days old even though the code was pushed today. Interesting.

**Pull request merged**